.github/workflows/build.yml

```diff
@@ -0,0 +1,76 @@
+name: Build
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+        branches:
+            - master
+jobs:
+    build:
+        name: PHP ${{ matrix.php-versions }} ${{ matrix.composer-args }}
+        strategy:
+            fail-fast: false
+            matrix:
+                operating-system: [ubuntu-latest]
+                php-versions: ['7.2', '7.3', '7.4']
+                composer-args: ['', '--prefer-lowest']
+                allow-failure: [false]
+                include:
+                    - operating-system: ubuntu-latest
+                      php-versions: '8.0'
+                      composer-args: '--ignore-platform-reqs --no-scripts'
+                      allow-failure: true
+                    - operating-system: ubuntu-latest
+                      php-versions: '8.0'
+                      composer-args: '--prefer-lowest --ignore-platform-reqs --no-scripts'
+                      allow-failure: true
+        runs-on: ${{ matrix.operating-system }}
+        continue-on-error: ${{ matrix.allow-failure }}
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2.3.3
+            - name: Setup PHP
+              uses: shivammathur/setup-php@2.5.0
+              with:
+                  php-version: ${{ matrix.php-versions }}
+                  extensions: mbstring
+                  coverage: pcov
+                  tools: cs2pr
+            - name: Setup problem matchers for PHPUnit
+              run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
+            - name: Get composer cache directory
+              id: composer-cache
+              run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+            - name: Cache dependencies
+              uses: actions/cache@v2
+              with:
+                  path: ${{ steps.composer-cache.outputs.dir }}
+                  key: php${{ matrix.php-versions }}-composer-${{ matrix.composer-args }}-${{ hashFiles('**/composer.json') }}
+                  restore-keys: php${{ matrix.php-versions }}-composer-${{ matrix.composer-args }}-
+            - name: Install Dependencies
+              run: |
+                  composer update --no-progress --no-suggest --prefer-stable --optimize-autoloader ${{ matrix.composer-args }}
+            - name: PHPStan
+              run: composer phpstan
+            - name: Code Style Check
+              env:
+                  PHP_CS_FIXER_IGNORE_ENV: true
```

Comment:
I had to add this for php-cs-fixer to run on PHP 8, they have an additional check in their executable that looks for this when the PHP version is out of their pre-defined range.